aries-core/package.json

```diff
@@ -42,10 +47,6 @@
     "build-storybook": "build-storybook",
     "release-stable": "babel-node ./tools/release-stable"
   },
-  "pre-commit": [
-    "lint-fix",
-    "test"
-  ],
```

Comment:
Is the `pre-commit` removed? I was actually thinking we should add "prettier" on pre-commit to help with code structure consistency.

Reply:
I would like to add Prettier as well.

We actually never had installed `pre-commit` as a `devDependency` and thus these config lines for `pre-commit` weren't being executed.

I am proposing that we use `husky` instead of `pre-commit`. They provide similar functionality, however I found `husky` to be easier to configure in a monorepo.

The scripts `lint-fix`, `test`, etc. will now be triggered by the `.huskyrc.js` file.